bouter/utilities.py

```diff
@@ -80,15 +80,37 @@ def log_dt(log_df, i_start=10, i_end=110):
 
 
 @jit(nopython=True)
-def fill_out_segments(tail_angle_mat, continue_curvature=0):
-    """ Fills out segments of tail trace
+def revert_segment_filling(fixed_mat, revert_pts):
```

Comment:
I suppose this is so the segments can be filled out with different interpolation paramters?

Reply:
yep